src/elements/Tag/Tag.stories.tsx

```diff
@@ -0,0 +1,47 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import { Tag, TagProps, TagType } from "./Tag";
+
+export default {
+    title: "Elements/Tag",
+    component: Tag,
+    args: {
+        label: "Label",
+    },
+    argTypes: {
+        type: {
+            options: Object.keys(TagType),
+            mappings: Object.values(TagType),
+            defaultValue: TagType.Suggested,
+            control: { type: "select" },
+        },
+    },
```

Comment:
```suggestion
    argTypes: {
        type: {
            options: Object.keys(TagType),
            mappings: Object.values(TagType),
            defaultValue: TagType.Suggested,
            control: { type: "select" },
        },
        onClick: { action: "onClick" },
    },
```

And then you can delete all `onClick` from the `.args = { ... }`

Reply:
The onClick is not set for all stories on purpose because the cursor style is different if onClick is undefined.

Reply:
It still works and shows the correct pointer since the type isn't `Selectable` or `SelectableWithFocus`, though.

Reply:
It's just that onClick is now always set even though it's not intended for the `Selectable` and `SelectableWithFocus` types. But you can leave it as it is, if you prefer this version.